earthquakes.py

```diff
@@ -11,47 +12,58 @@ def get_data():
     response = requests.get(
         "http://earthquake.usgs.gov/fdsnws/event/1/query.geojson",
         params={
-            'starttime': "2000-01-01",
-            "maxlatitude": "58.723",
-            "minlatitude": "50.008",
-            "maxlongitude": "1.67",
-            "minlongitude": "-9.756",
-            "minmagnitude": "1",
-            "endtime": "2018-10-11",
-            "orderby": "time-asc"}
+            'starttime': "2000-01-01", #beginning date of query
+            #Use a rectangle to cover the map (min-max lat and min-max long)
+            "maxlatitude": "58.723", #UK maximum latitude to query
+            "minlatitude": "50.008", #UK minimum latitude to query
+            "maxlongitude": "1.67", #Upper value of UK longitude to query
+            "minlongitude": "-9.756", #Lower value of UK longtitude to query
+            "minmagnitude": "1", #Looking for earthquakes with a higher magnitude than 1
+            "endtime": "2018-10-11", #End of date range we are looking into
+            "orderby": "time-asc"} #want time ascending dates
     )
 
     # The response we get back is an object with several fields.
     # The actual contents we care about are in its text field:
     text = response.text
-    # To understand the structure of this text, you may want to save it
-    # to a file and open it in VS Code or a browser.
-    # See the README file for more information.
-    ...
-
-    # We need to interpret the text to get values that we can work with.
-    # What format is the text in? How can we load the values?
-    return ...
+    
+    #write to a json file
+    with open('earthquakes_data.json', 'w') as earthquakes_data_out:
+        earthquakes_data_out.write(json.dumps(text,indent=4))
```

Comment:
Using indent is good idea and results in a more readable file. If the text you save doesn't look quite right, there is a related question on Moodle.